screens/login.html

```diff
@@ -151,7 +153,7 @@ <h3 class="head-title">Login</h3>
             <div class="pass-forget">
               <a href="#">Forget Password?</a>
```

Comment:
Should be Forgot Password. Also, red/pink color doesn't seem apt here. But,yeah its a personal choice.
Everything else looks good